resources/application-connector/templates/nginx-ingress-cert.yaml

```diff
@@ -4,6 +4,6 @@ data:
   "tls.key": {{ .Values.global.tlsKey | default "LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JSUV2UUlCQURBTkJna3Foa2lHOXcwQkFRRUZBQVNDQktjd2dnU2pBZ0VBQW9JQkFRQ3Vzd3MvVEY3eVpvTFkKdGFPSERvelUwZE9Mc3hheGFyZlUyNEROOUc2YTJvYmhXUG9Jd0RQeTd2bzB4Y2JXdy9idVpDZVp2bW5JejlPbQpCRXhNZUVMN0U2emdZR0RWZlZiSnhJc0pnL0FrM1I3bnVWcVdJc1lmUWFnNnYyclRvQ3JCbGZCREpaeXkvZE5GCm02UW5kNHFrbUNVR2lCRzg0amM4ZlRrWXU0bDBXOXBWbUpIUkY3bGlMdWpFMDlZbVdPbzN4Vm1lcUt2SDB4SWEKZ3h0and3Q2YxdXgxaXhjNTJiVlFkcFpkYi9XOFNjUDg0TkJGRlphY3BOaXJRN2NIZzk0cURXZy8zQnRnZjh4cwp3L3kya3NjaVYwYk5odHl6aGtocnpoU1dtSm1WYkpwL0lnUnM3Tm40VHRWc2gwbnRJWVNzZFJTWEpVSitIbksrCjRJKzl3ZThiQWdNQkFBRUNnZ0VBWHVKWjZvQkZWV3krQm92cERIS25wUE50N1lOUWdQNFFhR2pyOVhDY0hqa3cKWnZhWG51Y2RrQVdpR2NXTnNKbnc1UnU2c25OTUswaE5rTzdtcmVYbm8welZhV3lQUzRUdS9WbGxscjUxVU9CVQpaOUV4VlRmLzJYd2tyZnZIUHJmUVlTSlp3MjZMUkdBK1BxRkJNRHBxTCtsK3VxUnUxZU1iMHh2RVJoMmRrM2RsClNYSlRqUmJPd0VWU3dCTmg4Tk5JWnR3YlQrWGl3Wm5FcDFRTWF1MjBCZ2YxeXR6dTlIQUF2VkRVYkh2dFVYNGYKdVNHcFdWeVI3bjFzazRqdng0TmpzU0k3T0I5Ykg5Ulc4YUJ0L0piVVMxNTVRVWV1SUsrWjlvS09LbExRdDhIUwp5aTRHeVRtTndXdFQ2SzNtbUNQSTRwckNxYzh3Qzd1Q2lRKy9zbGl4OFFLQmdRRG9XOFBFc0FPQjVVUHpsQVpLClJhVStpSGJqTE13d3YwWWpDZXlXWk53bFNDUWc5U2g4bmVjMWpidXBMOSt2a2VXM1ZvWjNLSXBCWGhzME8xL0gKcmpnZjdSMWhROXExK0kyY1pzdmtPK0JZcnk4UGszSW9MS29rVnF1RVM3d2w0UmVFTThQNU0vV2VzRzNkd1grYwpHalkxSS90YjFLYlJBWmtma1JsZDFPZ1FhUUtCZ1FEQWVXOVVQaU9zYmY1clV1dkF0WFk0YXRBdkZlTmxCYmFOClgvZnNRa2h3UVZwWlJadFVKMGtwWWRlbFpqdnk5VVJ5Tk40RUl6bnF6VXYyakhYb3ZrajNMR3NIcHBUbzdyRkMKZUptZDU5RFB0cHNFZVh4My93RXMyVTBGUGlWU1BGOXMzemNPWHU0VGhnS1N5dkUrd0FJMXU0Ym84d3FIMFhZMgphZS8wTWJRUzR3S0JnSHVxVzBjVFpzcDdldTdvbVhCdUlNUm5YTjhMdUtFNUs5cjZMVERkL1V6YVdHQXlHcG80CmNCWFFtVkF5YXByR3k3d0lFUWphU3JyL212cnhuc2huc1BkTSs1WUl4eDZTYXN3UzZmRk83TmhHWklXbHM5d3QKNlYvdHdOdG9jM3Exa2c2S2ZvTGpaSlpRMENoMkU3VEJQcGlKemYzUjcvYk5JdkhxQjlZb3Vsb0JBb0dBV25lNQprRXMwMFZOU0JuZ1BQNkVMVkRGQWNTRkVwTE11dnZ5Z3QzZmdQSU03U0VLalk4YWphTVFJYkVIRlk4bE14ckY2CnVVOGxaZXZoREoyd3ZoalNpdnRodzFMVkl6Y3VNaVFsY0VjSnF0Z1Z2T0N0VkdtVHo5VERrdmRHNjRSZEJmdFYKRVhnbFJ4L3lEYlU5OTFTZ3RValRmS0xnREQ2ejVaZWRwWlhISWlrQ2dZRUFoL0lRMnhuS1hOUUZlL0d6cU1sYQpZZ3NqaWxhSkJlcW1qOTd6S2h5a3JvclZCTFhnU0ErSU5pMGpXY2pPUVZ5RmxNcjhBRFIvRm01NWNLTFFSOCtLCjNUOGFFUjJRRExTVVh5UWtkS0w0VVAvKzBXRmpSdGF6cktuV1JHZkl6Vmk2L253RXE2eVo4N3JQZGdwRmQzQmoKS0U0L1k0MFBSa21xY2wvc0FoUmdkK1U9Ci0tLS0tRU5EIFBSSVZBVEUgS0VZLS0tLS0K" }}
```

Comment:
Didn't we want to delete this deprecated value?

Reply:
Removed.
Reason: This certificate was old, issued for a domain we don't use anymore.
A valid "default" is already set in our installation scripts, so a default value in Helm charts will not be used anyway (it's always set)